### PR TITLE
More logging 

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] state log transfomers for chat,tracker,config,route-tree
- [x] logs some chat info. first 10 inbox items, last 30 messageMap items
- [x] filter out falsey / empty items

### CODE REVIEW

File: shared/reducers/tracker.js [hunk 1]
```diff
@@ -407,7 +407,7 @@ export default function (state: Constants.State = initialState, action: Action):
           ...state,
           pendingIdentifies: {
             ...state.pendingIdentifies,
-            [action.payload.username]: action.payload.pending,
+            [action.payload.username]: action.payload.pending ? true : undefined,
```
Comment (chrisnojima) [inline shared/reducers/tracker.js]: removes this from the logging

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

**Comment by cjb** [comment]
@chrisnojima flow errors

**Comment by chrisnojima** [comment]
- [x] base branch merged so this is off master now
- [x] fixed flow issue

@cjb

**Pull request merged**